### ISSUE TITLE
Implement value conversions that spread out over multiple columns

### ISSUE THREAD

**Issue by AndriySvyryd** [dotnet/efcore#31234]
Multi-column value conversions were initially tracked by #13947 as a way to enable value objects, but we decided to instead implement them using complex types.

This issue tracks using value conversions as an alternative to complex types.

#### Pros

- How properties map to columns is more flexible (e.g. 2 properties can be stored in one column or viceversa)

#### Cons

- Opaque to queries (i.e. individual properties participating in a multi-column value conversion can't be used in conditions or projections)

**Comment by mguinness** [comment]
From linked issue:

> Another area where this could be useful is storing [DateTimeOffset](https://mysqlconnector.net/troubleshooting/datetime-storage/#datetimeoffset-notes) into two fields for databases like MySQL which have no equivalent data type.

**Comment by RSoeborg** [comment]
Any ETA on this issue?

**Comment by roji** [comment]
@RSoeborg no, this issue is in the backlog and we don't currently have plans to work on it.
